net-im/telegram-desktop-bin/telegram-desktop-bin-4.0.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop optfeature xdg
+
+DESCRIPTION="Official desktop client for Telegram (binary package)"
+HOMEPAGE="https://desktop.telegram.org"
+SRC_URI="
+	https://github.com/telegramdesktop/tdesktop/archive/v${PV}.tar.gz -> tdesktop-${PV}.tar.gz
+	amd64? ( https://updates.tdesktop.com/tlinux/tsetup.${PV}.tar.xz )
+"
+
+LICENSE="GPL-3-with-openssl-exception"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+QA_PREBUILT="usr/bin/telegram-desktop"
+# current desktop-file-utils-0.26 does not understand Version=1.5
+# https://gitlab.freedesktop.org/xdg/desktop-file-utils/-/issues/59
+QA_DESKTOP_FILE="usr/share/applications/telegramdesktop.desktop"
```

Comment:
Oh yeah forgot about this, you can remove it. Current desktop-file-utils-0.26-r2 supports this now (was backported the -r2 revbump with a patch).